### ISSUE TITLE
Crash on trying to fill a barrel with liquid from a barrel

### ISSUE THREAD

**Issue by Craluminum2413** [anegostudios/VintageStory-Issues#1437]
**Game Version:** 1.16.0-pre.2
**Platform:** Windows
**Modded:** No
**SP/MP:** Singleplayer

### How to reproduce
1. Place a barrel
2. Fill the barrel with liquid
3. Put another barrel in active slot
4. Try to fill second barrel from the first
5. It crashes

### Logs
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.16.0-pre.2 (Unstable)12/02/2021 13:23:53: Critical error occurred
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Vintagestory.GameContent.BlockLiquidContainerBase.GetContent(ItemStack containerStack)
   at Vintagestory.GameContent.BlockLiquidContainerBase.GetNutritionProperties(IWorldAccessor world, ItemStack itemstack, Entity forEntity)
   at Vintagestory.API.Common.CollectibleObject.GetHeldTpUseAnimation(ItemSlot activeHotbarSlot, Entity forEntity)
   at Vintagestory.API.Common.EntityAgent.HandleHandAnimations()
   at Vintagestory.API.Common.EntityAgent.OnGameTick(Single dt)
   at Vintagestory.Client.NoObf.ClientSystemEntities.OnGameTick(Single dt)
   at Vintagestory.Common.EventManager.TriggerGameTick(Int64 ellapsedMilliseconds, IWorldAccessor world)
   at Vintagestory.Client.NoObf.ClientMain.MainRenderLoop(Single dt)
   at Vintagestory.Client.NoObf.ClientMain.MainGameLoop(Single deltaTime)
   at _9Q8fpX846Gx1pPUNQUDGlwGZBUB._SO6E7tLExeP0a8eb5s38wZGpGjr(Single )
   at _HRfDngnpBWTmEXiuwbRWkFKyJZs._yD9dPRZ9ZRfOdiu0s7HyEILMuoJ(Single )
   at _HRfDngnpBWTmEXiuwbRWkFKyJZs._vMYtOR2pAjMmy8Ygu8WqEmFNp0r(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) in C:\Users\Nexrem\Desktop\VintageStory\opentk\src\OpenTK\GameWindow.cs:line 479
   at OpenTK.GameWindow.DispatchRenderFrame() in C:\Users\Nexrem\Desktop\VintageStory\opentk\src\OpenTK\GameWindow.cs:line 454
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) in C:\Users\Nexrem\Desktop\VintageStory\opentk\src\OpenTK\GameWindow.cs:line 375
   at _aeznW7dUzRtbc6gGqfLLAAGPvUF._HH2M6AmcpFUMki4uDKOPMndAYft(_hFZgobIYoX0BrM3J1kokPKB9IqB , String[] )
   at _D1bjqjMU3djkJ14p5zl8JiWp0sG._HH2M6AmcpFUMki4uDKOPMndAYft(ThreadStart )
```

**Comment by Craluminum2413** [comment]
Noticed the same crash with click a barrel on a bucket

**Comment by Craluminum2413** [comment]
Fixed in [1.16.0-rc.5](Fixed: Game crashing when right clicking a barrel with a barrel in hands)
